geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/CreateAlterDestroyRegionCommandsDUnitTest.java

```diff
@@ -1091,6 +1086,126 @@ public void testDestroyRegionWithSharedConfig() {
     });
   }
 
+
+  final String PR_STRING = " package com.cadrdunit;"
```

Comment:
You could add a real class to a sub-package of this test in the source tree and then copy that class out to the .jar file. I'm not sure why you'd be making this so complex other than to avoid checking in a .jar (which is a disallowed binary).